packages/test/page-object/src/globals/pageObjects/verifyDOMIntegrity.js

```diff
@@ -41,4 +47,17 @@ export default function verifyDOMIntegrity() {
       throw new Error(message);
     }
   });
+
+  // Elements of `role="heading"` must have `aria-level` set
+  [].forEach.call(document.querySelectorAll('[role="heading"]'), element => {
+    const ariaLevel = +element.getAttribute('aria-level');
+
+    if (!(ariaLevel >= 1 && ariaLevel === ~~ariaLevel)) {
```

Comment:
Interesting... when would `ariaLevel === ~~ariaLevel` be `false`? ... `ariaLevel = true` is the only scenario I can think of for `!(ariaLevel >= 1 && ariaLevel === ~~ariaLevel)` to be `true`.

`!(true >= 1 && true === ~~true) => true`

Reply:
Hehe, `0.1`.

```
> value = 0.1
0.1
> value === ~~value
false
```

Reply:
A very large integer number could also return `false` for `bigInteger === ~~bigInteger`... such as an Unix epoch.